i18n/es.js

```diff
@@ -1314,7 +1319,7 @@
         "city": "Ciudad",
         "state": "Estado",
         "zip": "Código postal",
-        "poBox": "Mailing address is PO Box",
+        "poBox": "The Spanish text", // Will update later
```

Comment:
Are we still waiting on this? Just wanted to check if something missed getting pushed.

Reply:
Yup, please refer to https://github.com/episphere/connect/issues/994#issuecomment-2535919172